akkeeper-yarn/src/main/scala/akkeeper/yarn/LocalResourceNames.scala

```diff
@@ -19,6 +19,7 @@ private[akkeeper] object LocalResourceNames {
   val AkkeeperJarName = "akkeeper.jar"
   val UserJarName = "user.jar"
   val UserConfigName = "user_config.conf"
+  val LauncherConfigName = "application.conf"
```

Comment:
Shall we rename the constant name as well?

Reply:
:man_facepalming: :wink:

done